diff --git a/node_modules/react-native-maps/android/src/main/java/com/rnmaps/maps/MapMarker.java b/node_modules/react-native-maps/android/src/main/java/com/rnmaps/maps/MapMarker.java
https://github.com/react-native-maps/react-native-maps/pull/5103
index 016e023..1828622 100644
--- a/node_modules/react-native-maps/android/src/main/java/com/rnmaps/maps/MapMarker.java
+++ b/node_modules/react-native-maps/android/src/main/java/com/rnmaps/maps/MapMarker.java
@@ -703,4 +703,10 @@ public class MapMarker extends MapFeature {
         );
     }
 
+    @Override
+    protected void onLayout(boolean changed, int l, int t, int r, int b) {
+        super.onLayout(changed, l, t, r, b);
+        this.height = b-t;
+        this.width = r-l;
+    }
 }
